packages/core/src/awsService/cloudformation/ui/diffWebviewProvider.ts

```diff
@@ -137,6 +158,23 @@ export class DiffWebviewProvider implements WebviewViewProvider, Disposable {
                 </head>
                 <body>
                     <p>No changes detected for stack: ${this.stackName}</p>
+                    ${
+                        this.changeSetName &&
+                        this.changeSetStatus &&
+                        terminalChangeSetStatuses.includes(this.changeSetStatus)
+                            ? `
+                    <div class="deletion-button" style="margin: 10px 0; text-align: left; display: inline-block;">
+                        ${deletionButton}
+                    </div>
+                    <script>
+                        const vscode = acquireVsCodeApi();
+                        function deleteChangeSet() {
```

Comment:
think you need to move this out of the if block so it can be used by the button below (when there are non-zero changes)